test/com/google/enterprise/adaptor/sharepoint/SharePointAdaptorTest.java

```diff
@@ -5189,19 +5323,30 @@ public URLSegmentsExchange(String strURL, boolean getURLSegmentsResult,
     public final String lastItemIdOnPage;
     public final String getContentResult;
     public final WebServiceException exceptionToThrow;
-    
+    public final boolean useOnce;
```

Comment:
This entire code block is uncommented, with lots of nice boolean parameters, and should be refactored. For now, please add a comment here describing the semantics of useOnce. My understanding: "Signals that the ContentExchange will be called only once. Subsequent calls to MockSiteData.getContent will look for another registered ContentExchange."

Reply:
Done